server/api.go

```diff
@@ -84,6 +84,8 @@ func (p *Plugin) ServeHTTP(c *plugin.Context, w http.ResponseWriter, r *http.Req
 		p.updateSettings(w, r)
 	case "/api/v1/user":
 		p.getGitlabUser(w, r)
+	case "/api/v1/build":
```

Comment:
Can you please elaborate why an API endpoint is needed?

Reply:
I assumed like all the other commands this can also be integrated in web?

Reply:
Right now all slash commands are not supported via the Webapp. Let's remove it for now.